explorer/src/components/Staking/RegisteredSwitch.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC, useCallback, useMemo } from 'react'
+import { useViewStates } from 'states/view'
+
+export const RegisteredSwitch: FC = () => {
+  const { registeredOnly, setRegisteredOnly } = useViewStates()
+
+  const label = useMemo(
+    () => (registeredOnly ? 'Registered operator' : 'All operators'),
```

Comment:
`MyPositionSwitch` also has ["All operators"](https://github.com/subspace/blockexplorer/blob/08a648d43fd6bb168fe502eef59c2d3d3802d89e/explorer/src/components/Staking/MyPositionSwitch.tsx#L7) as an option which leads to a strange potential state:

![image](https://github.com/user-attachments/assets/f28bbda5-3391-46f0-be68-f53e18b0b1a3)

Maybe @DianaPertseva has ideas on better naming for either of these switches. For `MyPositionSwitch` "Staked only" isn't intuitive for me.

Reply:
This works for me

Reply:
I changed the switch for "My nominations" when active, the filter for status is now in the filters section